Update Docs.

Dushyantsingh-ds opened the pull request:
Script and spelling mistakes improved.

dnfadmin commented:
[![CLA assistant check](https://cla.dotnetfoundation.org/pull/badge/not_signed)](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=916) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=916) before we can accept your contribution.<br/><br/>:x: Dushyantsingh-ds [sign now](https://cla.dotnetfoundation.org/dotnet-foundation/website?pullRequest=916)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet-foundation/website?pullRequest=916) it.</sub>

Dushyantsingh-ds commented:
```
 dotnet run -- preview
 ```
Updated by

`
dotnet run -- preview
`
---------------
This website uses [Statiq Web]

Updated by

This website uses [Static Web]

Arhell reviewed (commented):
Thanks for your contribute, but what is in the readme is correct.
